### PR TITLE
PR for adding more additional ci testing

### PR DESCRIPTION
**Pull request by mmeier93**
## This PR does the following

- It adds additional CI testing for one specific GCTA flag
- It adds a testing profile for testing the ability for the pipeline to simulate Cohort Browser (CB) output data

## Details:

- Correct a few outdated header information in `test.config` (still had name of original config file)
- Re-ordered matrices in `various_flags_test` in `ci.yml` to match the order of flags in `nextflow.config`
- Added a matrix for `--gwas_simulation_replicates`
- Uploaded the `testdata/simulate_cb_output_config.yml` file to my own S3 bucket for now and made a `conf/cb.config` file. Then added a specific test inside `ci.yml`

## Comments:

- I originally tried adding more matrices to the `various_flags_test` to reflect as many optional flags as possible but this results in huge number of test (>50) so decided to keep just the most relevant (and if possible binary) flags for now.

## Testing:

- Latest GitHub Actions were successful
- Testing on CloudOS:
https://cloudos.lifebit.ai/app/jobs/5fc7f230cc96a20112a71ad3

(successfull but still get the weird non-100% complete I have encountered before, even though all results are there and all processes are successfull according to `stdout`)

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -22,8 +22,9 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        gwas_pheno_trait_type: ['binary', 'quantitative']
         effective_population_size: ['11418','11000']
```
Comment (cgpu) [inline .github/workflows/ci.yml:25]:
```Suggestion
        effective_population_size: [ 11418, 11000 ]
```

File: conf/cb.config [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+    num_participants = 40
+    simulate_vcf = true
+    simulate_plink = true
+    simulate_gwas_sum_stats = true
+    gwas_cases = 20
+    gwas_controls = 20
+    reference_1000G = "https://lifebit-featured-datasets.s3-eu-west-1.amazonaws.com/projects/gel/simulate/simulate-ci-testing/ALL_1000G_phase1integrated_v3_impute.tgz"
+    legend_for_hapgen2 = "https://lifebit-featured-datasets.s3-eu-west-1.amazonaws.com/projects/gel/simulate/simulate-ci-testing/all_leg_chr21.tar.gz"
+    simulate_cb_output = true
+    simulate_cb_output_config = "https://testdata-magda.s3-eu-west-1.amazonaws.com/simulate-ci-testing/simulate_cb_output_config.yml"
```
Comment (cgpu) [inline conf/cb.config:23]:
```Suggestion
    simulate_cb_output_config = "https://lifebit-featured-datasets.s3-eu-west-1.amazonaws.com/projects/gel/simulate/simulate-ci-testing/simulate_cb_output_config.yml"
```

File: .github/workflows/ci.yml [hunk 3]
```diff
@@ -22,8 +22,9 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
+        effective_population_size: [11418, 11000]
```
Comment (cgpu) [inline .github/workflows/ci.yml:25]:
```Suggestion
        effective_population_size: ['11418', '11000']
```

**Review by cgpu — APPROVED** [review]
Looks good 👍

File: conf/cb.config [hunk 4]
```diff
@@ -20,7 +20,7 @@ params  {
     reference_1000G = "https://lifebit-featured-datasets.s3-eu-west-1.amazonaws.com/projects/gel/simulate/simulate-ci-testing/ALL_1000G_phase1integrated_v3_impute.tgz"
     legend_for_hapgen2 = "https://lifebit-featured-datasets.s3-eu-west-1.amazonaws.com/projects/gel/simulate/simulate-ci-testing/all_leg_chr21.tar.gz"
     simulate_cb_output = true
-    simulate_cb_output_config = "https://testdata-magda.s3-eu-west-1.amazonaws.com/simulate-ci-testing/simulate_cb_output_config.yml"
+    simulate_cb_output_config = "https://lifebit-featured-datasets.s3-eu-west-1.amazonaws.com/projects/gel/simulate/simulate-ci-testing/simulate_cb_output_config.yml"
```
Comment (cgpu) [inline conf/cb.config:23]:
```Suggestion
    simulate_cb_output_config = "https://testdata-magda.s3-eu-west-1.amazonaws.com/simulate-ci-testing/simulate_cb_output_config.yml
```

Comment (cgpu) [inline conf/cb.config:23]:
```Suggestion
    simulate_cb_output_config = "https://testdata-magda.s3-eu-west-1.amazonaws.com/simulate-ci-testing/simulate_cb_output_config.yml"
```

File: nextflow.config [hunk 5]
```diff
@@ -8,7 +8,7 @@
 // Define image used by pipeline
 
 docker.enabled = true
-process.container = 'lifebitai/simulate:latest'
+process.container = 'lifebitai/simulate:1.0dev-8cfe5ae'
```
Comment (cgpu) [inline nextflow.config:11]:
```Suggestion
process.container = 'quay.io/lifebitai/simulate:5c999c5'
```

**Comment by mmeier93** [comment]
Thanks 👍

**Pull request merged**